src/Components/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import '../Styles/App.scss';
+import Home from './Home';
+import Pet from './Pet';
+import PetApplications from './PetApplications';
+import Error from './Error';
+
+
+
+function App() {
+  return (
+    <div className="App">
+      <Routes>
```

Comment:
First time setting up Router with version 6. Discussions about nesting routes. We want to do more research to figure out if this will be best as is or if we will need to change.